saviSimulation/savi/jason_processing/UxV.java

```diff
@@ -78,7 +78,7 @@ public void draw(PVector position) {
 		//draw image
 		simulator.shape(image, position.x, position.y, pixels, pixels);
 		//show height lower and upper
-		simulator.text(Double.toString(position.z+(this.pixels/2))+"\n"+Double.toString(position.z-(this.pixels/2)), position.x, position.y);
+		simulator.text(Double.toString(getBehavior().compasAngle)+"\n"+Double.toString(position.z-(this.pixels/2)), position.x, position.y);
```

Comment:
Again, for debugging. Although this was pretty helpful for debugging issues with the bearing... may be useful information to keep.

Reply:
ideally all the debug stuff could be switched on/ off with a static flag "debug mode" (config or just manually set somewhere)

Reply:
Definitely had the same thoughts when debugging the behaviour. This would be awesome!